processor/transformer/manager.go

```diff
@@ -0,0 +1,34 @@
+package transformer
+
+import (
+	"context"
+	"errors"
+
+	"github.com/rudderlabs/rudder-server/processor/types"
+)
+
+type ServiceClient interface {
+	SendRequest(ctx context.Context, clientEvents []types.TransformerEvent, batchSize int) types.Response
+}
+
+type CommunicationManager struct {
+	clients map[string]ServiceClient
+}
+
+func NewCommunicationManager() *CommunicationManager {
+	return &CommunicationManager{
+		clients: make(map[string]ServiceClient),
+	}
+}
+
+func (m *CommunicationManager) RegisterService(name string, client ServiceClient) {
+	m.clients[name] = client
+}
+
+func (m *CommunicationManager) GetServiceClient(name string) (ServiceClient, error) {
```

Comment:
looks like we're not using this and continuing using the older transformer client(one client for all)